boards/px4/fmu-v4/init/rc.board_sensors

```diff
@@ -5,14 +5,6 @@
 
 board_adc start
 
-# We know there are sketchy boards out there
-# as chinese companies produce Pixracers without
-# fully understanding the critical parts of the
-# schematic and BOM, leading to sensor brownouts
-# on boot. Original Pixracers following the
-# open hardware design do not require this.
-pwm_out sensor_reset 50
```

Comment:
This doesn't appear to be necessary with the current state of the drivers (tested on a shady pixracer clone).